traitsui/testing/api.py

```diff
@@ -78,13 +84,13 @@
     Slider
 )
 
+# Advanced usage
 from .tester.target_registry import TargetRegistry
 
-from .tester.query import (
-    DisplayedText,
-    IsChecked,
-    IsEnabled,
-    SelectedText
+# Exceptions
+from .tester.exceptions import (
+    Disabled,
+    InteractionNotSupported,
+    LocationNotSupported,
+    TesterError
```

Comment:
On the one hand, I agree that having the import order matching the docstring helps detecting missing entries in the docstring, so it is useful. On the other hand, it goes against the usual convention of alphabetically ordering imports.

If you'd like to keep the new ordering here, because this is out of the ordinary of alphabetical order, it might be worth a comment to explain the grouping matches the docstring. In any case, the new comments like `# Exceptions` are good!